src/assets/icons/twitter.svg

```diff
@@ -1,31 +1,13 @@
-import React from 'react';
-import styled from 'styled-components';
+<svg
 
-interface TwitterProps {
-  color?: string;
-}
-
-export const IconTwitter: React.FC<TwitterProps> = (props) => (
-  <svg
-    width="27"
-    height="22"
     viewBox="0 0 27 22"
```

Comment:
a bounding box of `0 0 24 24` is required to properly align. You can't just change the numbers as the underlying vectors don't fit the 1:1 aspect ratio.  I think it's best to ask Verena for help. Usually you create a square of 24x24px in your favourite illustration app and insert and scale the icons so it's properly aligned.

This issue happens in the other icons too.

![Screenshot 2022-02-25 at 09 03 16](https://user-images.githubusercontent.com/1701755/155678201-bf1bda5a-e5ac-4b42-9bca-b1ebe8bc4b37.png)